serverless/client/client.go

```diff
@@ -386,3 +389,38 @@ func (lst *LogStateTracker) Update(ctx context.Context) error {
 	lst.LatestConsistentRaw, lst.LatestConsistent = cRaw, *c
 	return nil
 }
+
+// CheckConsistency is a wapper function which simplifies verifying consistency between two or more checkpoints.
+func CheckConsistency(ctx context.Context, h hashers.LogHasher, f Fetcher, cp []log.Checkpoint) error {
```

Comment:
I wonder if having `cp` being variadic would be useful in some cases?

Reply:
Hmm, dunno, do you have any cases like that in mind?